app/frontend/src/features/auth/AuthPage.tsx

```diff
@@ -47,61 +62,145 @@ const useStyles = makeStyles((theme) => ({
     height: "50vh",
     justifyContent: "center",
     textAlign: "center",
+    [theme.breakpoints.up("md")]: {
+      height: "100%",
+      margin: "0 auto",
+      width: "960px",
+    },
+  },
+  desktopNavigation: {
+    display: "flex",
+  },
+  introduction: {
+    color: "#ffffff",
+    textAlign: "left",
+    width: "742px",
+  },
+  loginLink: {
+    border: `1px solid ${theme.palette.primary.main}`,
+    borderRadius: "4px",
+    color: "#ffffff",
+    fontWeight: 500,
+    fontSize: "20px",
+    marginRight: theme.spacing(3),
+    padding: theme.spacing(1, 0),
+    textAlign: "center",
+    textDecoration: "none",
+    width: "88px",
   },
-  navigation: {
+  mobileNavigation: {
     display: "flex",
     justifyContent: "space-around",
-    marginTop: theme.spacing(3),
+    marginTop: theme.spacing(3, 0),
+  },
+  signupLink: {
```

Comment:
refactor this and `loginLink` to use a shared class and then only have the differences separately.